src/rantanplan/core.py

```diff
@@ -13,6 +13,8 @@
 
 from spacy.tokens import Doc
 
+from rantanplan.alternative_syllabification import SYLLABIFICATOR_FOREIGN_WORDS_DICT
```

Comment:
Not `from .alternative_syllabification import SYLLABIFICATOR_FOREIGN_WORDS_DICT`?

Reply:
Yes, changed it